selfdrive/car/subaru/carcontroller.py

```diff
@@ -32,13 +38,22 @@ def update(self, CC, CS, now_nanos):
       new_steer = int(round(apply_steer))
       apply_steer = apply_driver_steer_torque_limits(new_steer, self.apply_steer_last, CS.out.steeringTorque, self.p)
 
+      apply_steer_req = 1
+
       if not CC.latActive:
         apply_steer = 0
+        apply_steer_req = 0
+
+      if self.CP.carFingerprint in STEER_RATE_LIMITED:
+        # Steering rate fault prevention
+        self.steer_rate_counter, apply_steer_req = \
+          common_fault_avoidance(abs(CS.out.steeringRateDeg) > MAX_STEER_RATE, apply_steer_req,
+                                 self.steer_rate_counter, MAX_STEER_RATE_FRAMES)
 
       if self.CP.carFingerprint in PREGLOBAL_CARS:
         can_sends.append(subarucan.create_preglobal_steering_control(self.packer, self.frame // self.p.STEER_STEP, apply_steer, CC.latActive))
       else:
-        can_sends.append(subarucan.create_steering_control(self.packer, apply_steer, CC.latActive))
+        can_sends.append(subarucan.create_steering_control(self.packer, apply_steer, apply_steer_req))
```

Comment:
```suggestion
        apply_steer_req = CC.latActive
        if self.CP.carFingerprint in STEER_RATE_LIMITED:
          # Steering rate fault prevention
          self.steer_rate_counter, apply_steer_req = common_fault_avoidance(abs(CS.out.steeringRateDeg) > MAX_STEER_RATE,
                         apply_steer_req, self.steer_rate_counter, MAX_STEER_RATE_FRAMES)
        can_sends.append(subarucan.create_steering_control(self.packer, apply_steer, apply_steer_req))
```

Formatting is probably wrong